packages/react/src/runtime/ax.ts

```diff
@@ -27,10 +27,14 @@ const ATOMIC_GROUP_LENGTH = 5;
  *
  * @param classes
  */
-export default function ax(classNames: (string | undefined | false)[]): string | undefined {
+export default function ax(
```

Comment:
Could you please do a benchmark for the function? I don't think there's anything controversial here but important to be careful

Reply:
- `ax` on Master branch: 486,792 ops/sec
- `ax` on this branch: 212,283 ops/sec
- `ax` with compressed class names on this branch: 261,141 ops/sec

Based on the benchmark, this PR makes `ax` 2 times slower due to additional `if Array.isArray()`. Hopefully the performance gain from compressing the classnames outweighs the loss here.
Note `ax([['aaaa', 'b'], ['aaaa', 'c']])` is slightly more performant than `ax(['_aaaabbbb', '_aaaacccc'])`. I think it's because the former skips `.slice`.

Reply:
Awesome work!